lib/main.dart

```diff
@@ -29,6 +30,10 @@ void main() async {
   WidgetsFlutterBinding
       .ensureInitialized(); //required to get sqlite filepath from path_provider before UI has initialized
   await openDatabase();
+  // Demo using SQLite Full-Text Search with PowerSync.
+  // See https://docs.powersync.com/usage-examples/full-text-search for more details
+  await configureFts(db);
```

Comment:
I'd move this to inside the `openDatabase()` function in `powerync.dart`.

Reply:
Is that so we don't confuse new users with this additional functionality?